src/sentry/web/api.py

```diff
@@ -587,6 +587,53 @@ def process(self, request, project, key, auth, helper, data, attachments=None, *
                              key, remote_addr, helper, attachments)
 
 
+class EventAttachmentStoreView(StoreView):
+
+    def post(self, request, project, event_id, **kwargs):
+        if not features.has('organizations:event-attachments',
+                            project.organization, actor=request.user):
+            raise APIForbidden("Event attachments are not enabled for this organization.")
+
+        if len(request.FILES) == 0:
+            return HttpResponse(status=400)
+
+        for name, uploaded_file in six.iteritems(request.FILES):
+            file = File.objects.create(
+                name=uploaded_file.name,
+                type='event.attachment',
+                headers={'Content-Type': uploaded_file.content_type},
```

Comment:
Multipart uploads do not have content types set typically. I just realized we have the same bug on the minidump upload but we really shoudl guess the mimetype from the file extension here with `mimetypes.guess_type` unless the mimetype is actually set.

Reply:
👍 I will take this in a follow-up.